src/applications/mhv/medical-records/util/constants.js

```diff
@@ -143,7 +143,7 @@ export const pageTitles = {
   CARE_SUMMARIES_AND_NOTES_PAGE_TITLE:
     'Care Summaries And Notes - Medical Records | Veterans Affairs',
   VACCINES_PAGE_TITLE: 'Vaccines - Medical Records | Veterans Affairs',
-  ALLERGIES_PAGE_TITLE: 'Allergies - Medical Records | Veterans Affairs',
+  ALLERGIES_PAGE_TITLE: 'Allergies and Reactions | Veterans Affairs',
```

Comment:
According to the ticket, this should be `<title>Allergies and reactions - Medical Records | Veterans Affairs</title>`

Reply:
Corrected.